tree-sitter-stack-graphs/src/cli/test.rs

```diff
@@ -209,12 +209,26 @@ impl TestArgs {
             .with_context(|| format!("Loading builtins into {}", test_path.display()))?;
         let mut globals = Variables::new();
         for test_fragment in &test.fragments {
-            let fragment_path = Path::new(test.graph[test_fragment.file].name()).to_path_buf();
-            if lc.matches_file(&fragment_path, Some(&test_fragment.source)) {
+            if let Some(fa) = test_fragment
+                .path
+                .file_name()
+                .and_then(|f| lc.special_files.get(&f.to_string_lossy()))
```

Comment:
Right, here is where that `get` method is being called, and I _think_ that it should be safe to borrow `lc`'s reference count instead of having to having to inc and dec the reference count just for this `if` statement.

If it's too thorny to make the borrow checker happy please feel free to disregard.